### PR TITLE
chore: update document for cloud

### PR DESCRIPTION
**Pull request by xieydd**
Add document for authorization, API key and backup&restore.

### CODE REVIEW

File: src/cloud/connect/connect-with-psql.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ We provide two types of connection endpoints:
 - Super User Endpoint: This connection string is used to connect to the cluster as a superuser(postgres). You can use this connection string to create databases, users, and manage the cluster.
 - Vector User Endpoint: This connection string is used to connect to the cluster as a vector user. You can use this connection string to store and query vector data.
 
+If you enable pooler connection, you can get the pooler connection endpoint via clicking the `Show Pooler Endpoint` button.
```
[the remark this answers is not in the document]
Comment (xieydd) [reply src/cloud/connect/connect-with-psql.md]: fixed

File: src/cloud/manage/apikey.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Create an API Key
+
+To create an API key, follow these steps:
+- Navigate to the **API Key** section in the left-hand menu.
+- Click the **Create API Key** button.
+- Enter a name for the API key.
+- Select the role for the API key.
+    - **Owner**: The owner has full access to the project cluster.
+    - **Member**: The member has read-only access to the project cluster, but you can specify clusters full access.
```
[the remark this answers is not in the document]
Comment (xieydd) [reply src/cloud/manage/apikey.md]: fixed

File: src/cloud/manage/apikey.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+:::
+
+## List of API Keys
+
+To view the list of API keys, follow these steps:
+- Navigate to the **API Key** section in the left-hand menu.
+- You can view the list of API keys with their respective roles.
+
+::: warning
+If you are not the owner of the project, you will not be able to view the list of API keys.
```
[the remark this answers is not in the document]
Comment (xieydd) [reply src/cloud/manage/apikey.md]: fixed

**Review by cutecutecat — APPROVED** [review]

File: src/cloud/manage/backup-restore.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- Enter the backups details page of a cluster.
+- Click the `Actions` button and select `Restore`
+    ![](../images/restore-backup.png)
+- Select the server resource that you want to restore to.
+
+![alt text](../images/restore-backup-cluster.png)
+
+
+::: warning
+The cluster you are restoring to should have the same images and database as the backup or cluster.
```
Comment (gaocegege) [inline src/cloud/manage/backup-restore.md:76]: Here

Comment (xieydd) [reply src/cloud/manage/backup-restore.md]: fixed

**Review by gaocegege — APPROVED** [review]

**Pull request merged**